lib/fizzy/cxx20/bit.hpp

```diff
@@ -7,6 +7,10 @@
 #include <cstdint>
 #include <type_traits>
 
+#if __cplusplus > 201703L
```

Comment:
```suggestion
#if __has_include(<version>)
```
https://en.cppreference.com/w/cpp/preprocessor/include

Reply:
Nice!